docs/quick_start/update.rst

```diff
@@ -0,0 +1,41 @@
+.. Images
+.. |alert| image:: ../images/alert.jpg
+
+Package Updates
+===============
+
+Fledge will periodically check for updates to the various packages that are installed. If updates are available then this will be indicated by a status indicating on the bar at the top of the Fledge GUI.
+
++---------+
+| |alert| |
++---------+
+
+Clicking on the *bell* icon will display the current system alerts, including the details of the packages available to be updated.
```

Comment:
Have you noticed periodic task upgrade alert is generating per each package?

For development purpose (i.e nightly or develop) that would makes sense But for **Production** it won't make any sense to get alerts per package. Because whenever we release new version of fledge, we released other packages as well with condition `>=<fledge.version>` ! 

**For example:**

Let say, User have installed fledge version 2.3.1 (_in which alert periodic feature is there_) along with 
10 packages.

Now, on the periodic task run we found the new upgrades (let say 2.3.2 version is available). And periodic task will generate 10 alerts for the same stuff.
```
{
"alerts": [
{
"key": “fledge-gui",
"message": "A newer version of fledge-gui is available for upgrade",
"urgency": "Normal",
"timestamp": "2024-03-12 07:30:55.232"
},
{
"key": “fledge”,
"message": "A newer version of fledge is available for upgrade",
"urgency": "Normal",
"timestamp": "2024-03-12 07:30:55.238"
},
{
"key": “fledge-south-sinusoid”,
"message": "A newer version of fledge-south-sinusoid is available for upgrade",
"urgency": "Normal",
"timestamp": "2024-03-12 07:30:55.238"
},

…
…
{
"key": “fledge-service-notification”,
"message": "A newer version of fledge-service-notification is available for upgrade",
"urgency": "Normal",
"timestamp": "2024-03-12 07:30:55.238"
},
…
…

]
}
```
I believe for Production it does not make any sense to generate and show 10 alerts to users. Because when user will try to upgrade fledge either via `curl -X PUT http://localhost:8081/fledge/update` or on command line `apt install fledge` the fledge will automatically upgrade along with the dependent packages.

I believe, periodic task should only generate a single alert for version upgrade:
```
{
"key": “upgrades”,
"message": "A newer version 2.3.2 of fledge is available for upgrade",
"urgency": "Normal",
"timestamp": "2024-03-12 07:30:55.238"
}
```
OR 
`message: "A newer version of fledge, fledge-gui, fledge-south-sinusoid, fledge-service-notification, ....  is available for upgrade",`

Reply:
Yes, I think the update checker needs to be changed as it is not what was intended.

Reply:
Please create a JIRA as fit!

Reply:
I see FOGL-8579, Thanks!